### PR TITLE
License preference, per discussion with legal

### PR DESCRIPTION
**Pull request by brianwarner**
From a discussion on the Board, I checked with OpenJS Foundation legal counsel on the preferred license, all other things being equal. While we have a number of options in the IP policy, Apache 2.0 seems to be the preferred option if there are no other reasons to choose something else.

Signed-off-by: Brian Warner <brian@bdwarner.com>

### CODE REVIEW

File: proposals/stage-2/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md
Comment (ljharb) [inline proposals/stage-2/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md:76]: can i ask why? MIT is the *resounding* majority in the JS ecosystem; apache-2 is quite rare there (it's more common in python)

**Comment by ljharb** [comment]
In other words, I'd say for a foundation purporting to be for JS, "not MIT" is to an *exceeding* degree what needs the burden of proof.

File: proposals/stage-2/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md
Comment (eemeli) [reply proposals/stage-2/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md:76]: My presumption would be that it's due to the explicit patent grants included in Apache 2, unlike the other supported licenses. But this definitely does need some clear explanation around it arguing the case.

Comment (brianwarner) [reply proposals/stage-2/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md:76]: That's right on the explicit patent grants, @eemeli. 

That said, I would actually recommend against arguing any case here. The idea here was not to get too prescriptive, but rather to provide gentle guidance to someone who genuinely doesn't have an opinion, so they don't feel they are choosing from random. If there's even the slightest preference otherwise, such as what you're saying @ljharb, then they're definitely empowered by the IP policy to choose what they prefer. But since that's already written down (and following @tobie's general lead on keeping things simple) I'm not sure it makes sense to get into it here.

Thoughts?

Comment (dylans) [reply proposals/stage-2/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md:76]: > can i ask why? MIT is the _resounding_ majority in the JS ecosystem; apache-2 is quite rare there (it's more common in python)

My experience is that in general lawyers and enterprises prefer Apache 2, with a second choice being the new BSD, and a third choice being MIT. Generally the reverse order is true for JS open-source maintainers. For most practical purposes the licenses are roughly the same and generally considered equal, except that Apache is explicit around patent rights.

The primary reason JS devs historically chose MIT over the years is that it is terse, and in the early days the necessary approach was to require the license at the top of each file. The MIT license was short enough to do that, whereas the Apache license was a few pages and back before bundlers/build tools were popular, embedding the license would add a significant increase to file size. Either that or it's a function of tl;dr;. And from there as an ecosystem we typically just copy the license used by other projects we respect.

Comment (tobie) [reply proposals/stage-2/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md:76]: This feels like an odd context to express that preference. If the Board/Foundation feels strongly about Apache 2 over MIT, why not to some advocacy and evangelism on the blog and share the arguments behind the suggestion? This feels a bit underhanded to me.

Comment (ljharb) [reply proposals/stage-2/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md:76]: It’s also not a universal preference - the lawyers at a previous employer did not prefer Apache over mit at all, and in fact their strong preference was “do whatever the relevant ecosystem prefers”.

Comment (brianwarner) [reply proposals/stage-2/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md:76]: I'd just maybe recommend we take a step back, as the point was not that the board feels strongly about choosing between one or another pre-approved license. The discussion really was just "If a contributor is creating something new and doesn't have a license preference, which of the four options would be recommended so they aren't choosing at random?" The key thing is that this just establishes a default for this particular program, and if there's any reason to choose another license they're able to do so under the IP policy (as with OpenJS Foundation projects).

The main takeaway is that the board felt this was a missing bit of info, so the action was to check with counsel and add a line to the proposal so they can vote on it.

Comment (ljharb) [reply proposals/stage-2/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md:76]: Suggesting a default makes total sense :-) i commented because i think suggesting any default for a JS project besides "MIT" is confusing, since MIT (or ISC) is by a *large* margin the most common license used in the JS ecosystem.

File: proposals/stage-2/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md
Comment (ljharb) [inline proposals/stage-2/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md:76]: it'd be great if this included *why* the foundation recommends that.

**Review by MylesBorins — APPROVED** [review]
LGTM

**Review by kborchers — APPROVED** [review]

**Review by tobie — APPROVED** [review]
I like the language in that last iteration. It is much better. I still believe this would be better spelled out in a broader context (maybe the IP policy), but I don't want to block on this.

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by sendilkumarn — APPROVED** [review]

**Review by dmethvin — APPROVED** [review]

**Review by eemeli — APPROVED** [review]

**Review by molant — APPROVED** [review]

**Review by joesepi — APPROVED** [review]

**Pull request merged**
